Client/mods/deathmatch/logic/luadefs/CLuaCameraDefs.h

```diff
@@ -24,7 +24,7 @@ class CLuaCameraDefs : public CLuaDefs
     LUA_DECLARE(GetCameraTarget);
     LUA_DECLARE(GetCameraInterior);
     LUA_DECLARE(GetCameraGoggleEffect);
-    LUA_DECLARE(GetCameraShakeLevel);
+    LUA_DECLARE(GetCameraDrunkLevel);
```

Comment:
Consider using the new argument parser for this.

Reply:
Sure mate, tmrw